framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/HACepScenarioImpl.java

```diff
@@ -125,16 +135,31 @@ protected void deployKieDeployments() {
             throw new RuntimeException("File with HACEP role binding can not be found: " +
                                                roleBindingYamlFile.getAbsolutePath());
         }
+        logger.info("Creating role for HACEP from file: {}", roleYamlFile.getAbsolutePath());
         project.createResourcesFromYamlAsAdmin(roleYamlFile.getAbsolutePath());
+        logger.info("Creating service account for HACEP from file: {}", serviceAccountYamlFile.getAbsolutePath());
         project.createResourcesFromYamlAsAdmin(serviceAccountYamlFile.getAbsolutePath());
+        logger.info("Creating role binding for HACEP from file: {}", roleBindingYamlFile.getAbsolutePath());
         project.createResourcesFromYamlAsAdmin(roleBindingYamlFile.getAbsolutePath());
 
-        project.createResourcesFromYamlAsAdmin(OpenShiftConstants.getHaCepResourcesList());
+        final String imageURL = buildHACEPImage();
+        final File haCepDeploymentYamlFile = new File(haCepSourcesDir, SOURCES_FILE_HACEP_DEPLOYMENT);
+        final String haCepDeploymentYaml = patchHACEPDeploymentFile(haCepDeploymentYamlFile, imageURL);
```

Comment:
I don't know how exactly the deployment file looks like, but couldn't be set the image url as parameter?

Reply:
I would prefer to keep it as it is. At least for now. It uses the same setup as other parts of HA-CEP automation, which expects unpacked folder with HACEP distribution in some folder. It is probably possible to allow urls, but this will affect much more parts of automation that it would be better to have a separate PR for it. Also there is Docker build running from folder with HACEP distribution using oc command.